great_expectations/checkpoint/actions.py

```diff
@@ -845,16 +876,25 @@ class StoreValidationResultAction(ValidationAction):
         TypeError: validation_result_id must be of type ValidationResultIdentifier or GeCloudIdentifier, not {}.
     """
 
+    type: Literal["store_validation_result"] = "store_validation_result"
+
+    class Config:
+        arbitrary_types_allowed = True
+
+    _target_store: Store = PrivateAttr()
+
```

Comment:
This action still takes a `data_context` as an `__init__` arg. It seems like it should take a `data` argument. This should this be updated. It seems it was like this before and can be updated in a separate PR. If not as a fast follow PR to this one, it should be ticketed and added to this sprint.

Reply:
I'll be deleting this action once we get validation definitions to store their results. These changes are just to ensure tests pass for now